<?xml version="1.0"?>
<launch>
	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="beer::link"/>
		<arg name="frame_id" value="beer"/>
	</include>

	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="table::link"/>
		<arg name="frame_id" value="table_1"/>
	</include>

	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cafe_table::link"/>
		<arg name="frame_id" value="table_2"/>
	</include>
</launch>
